src/components/app/shared/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import PandalCard from './PandalCard';
+import { TbLocationFilled } from 'react-icons/tb';
+import { IoMdTrendingUp } from 'react-icons/io';
+
+const Card = () => {
+  const [activeCard, setActiveCard] = useState<'trending' | 'nearme'>('trending');
+  return (
+    <>
+      <div className="flex flex-row gap-2">
+        <button
+          className={`p-2 px-3 font-work rounded-full bg-[${activeCard === 'trending' ? '#fff' : '#e6dfcf'}]`}
```

Comment:
use `cn` for `className` conditionals. e.g - https://github.com/Pujo-Atlas-Kolkata/PujoAtlasKol-Web/blob/dev/src/components/app/shared/BottomNav.astro